src/components/progress-tracker/progress-tracker.test.js

```diff
@@ -169,4 +181,135 @@ context("Tests for ProgressTracker component", () => {
       }
     );
   });
+
+  describe("should checkt accessibility tests for progress tracker component", () => {
```

Comment:
A typo here"describe("should **check** accessibility tests for progress tracker component", () => {"